test/events.js

```diff
@@ -122,3 +122,43 @@ test('Emit event when broker closed', function (t) {
   })
   broker.close()
 })
+
+test('Test backpressure aedes published function', { timeout: 500 }, function (t) {
+  t.plan(1)
+
+  const broker = aedes({
+    published: function (packet, client, done) {
+      if (client) {
+        t.pass('published')
+        setTimeout(done, 1000)
```

Comment:
I don't understand how this timeout relates to the timeout in the actual test.

Reply:
this `setTimeout` is to stimulate a backpressure on published function. The whole test timeout is set less than 1000 to let the test plan end with 1 passed result.

Reply:
How long will the test last?

Reply:
500